.ci/chatops-commands/ci-problems-analyze.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    >&2 echo "Usage: $0 PR_NUMBER"
+    exit 1
+fi
+
+set -e
+
+OWNER_NAME="camunda"
+REPO_NAME="camunda"
+PR_NUMBER="$1"
+PR_HEAD_SHA=$(gh api -X GET "repos/$OWNER_NAME/$REPO_NAME/pulls/$PR_NUMBER" --jq '.head.sha')
```

Comment:
NABD: Feel free to ignore this comment, but In this repository, sometimes the bash variables in scripts are enclosed in {} and other not. I know both ways are correct but I just wanted to point out that in infra-core most of the times we enclose these variables in {} and I personally find it cleaner.